nvc.1

```diff
@@ -501,6 +505,58 @@ hierarchy coverage report in HTML format, run:
 .Bd -literal -offset indent
 $nvc -c --merge=merged.covdb --report=<path_to_folder_for_html_report> first.covdb second.covdb third.covdb ...
```

Comment:
This was from the previous patch but this line is really long and wraps if you view it with `man` in a terminal. Break it over two lines like:

```
.Bd -literal -offset indent
$ nvc -c --merge=merged.covdb --report=<path_to_folder_for_html_report> \\
        first.covdb second.covdb third.covdb ...
.Ed
```

Reply:
Done.